apis/v1/httproute_types.go

```diff
@@ -612,9 +612,14 @@ type HTTPHeaderMatch struct {
 	Name HTTPHeaderName `json:"name"`
 
 	// Value is the value of HTTP Header to be matched.
+	// <gateway:experimental:description>
+	// Must consist of printable US-ASCII characters, optionally separated
+	// by single tabs or spaces. See: https://tools.ietf.org/html/rfc7230#section-3.2
+	// </gateway:experimental:description>
 	//
 	// +kubebuilder:validation:MinLength=1
 	// +kubebuilder:validation:MaxLength=4096
+	// <gateway:experimental:validation:Pattern=`^[!-~]+([\\t ]?[!-~]+)*$`>
```

Comment:
Is the escaping of the `\\t` correct here? I assume that's meant to match a tab?

This seems like one of those things that unless we're sure could be a really subtle bug, because if we get the escaping wrong, this will match `\` and `t` as well as ` `.

Reply:
Good catch, transitioning from double quotes to backticks often causes issues 😅